lib/saucelabs/start_tunnel.sh

```diff
@@ -11,14 +11,14 @@ set -e
 # Curl and run this script as part of your .travis.yml before_script section:
 # before_script:
 #   - curl https://gist.github.com/santiycr/5139565/raw/sauce_connect_setup.sh | bash
-SC_VERSION="4.5.1"
+SC_VERSION="4.5.2"
 CONNECT_URL="https://saucelabs.com/downloads/sc-$SC_VERSION-linux.tar.gz"
 CONNECT_DIR="/tmp/sauce-connect-$RANDOM"
 CONNECT_DOWNLOAD="sc-$SC_VERSION-linux.tar.gz"
 
-CONNECT_LOG="$LOGS_DIR/sauce-connect"
-CONNECT_STDOUT="$LOGS_DIR/sauce-connect.stdout"
-CONNECT_STDERR="$LOGS_DIR/sauce-connect.stderr"
+# We don't want to create a log file because sauceconnect always logs in verbose mode. This seems
+# to be overwhelming Travis and causing flakes when we are cat-ing the log in "print_logs.sh"
+CONNECT_LOG="/dev/null"
```

Comment:
Interesting!

Reply:
TBH, I don't know if it was actually affecting flakiness. I was just porting bits that _looked_ useful from Angular PRs. (This came from angular/angular#27657.)